packages/compiler-cli/src/ngtsc/diagnostics/src/error_code.ts

```diff
@@ -161,6 +161,15 @@ export enum ErrorCode {
   INJECTABLE_DUPLICATE_PROV = 9001,
 }
 
+/**
+ * @internal
+ */
+export const HAS_GUIDE = new Set([
+  ErrorCode.DECORATOR_ARG_NOT_LITERAL, ErrorCode.PARAM_MISSING_TOKEN,
+  ErrorCode.SCHEMA_INVALID_ELEMENT, ErrorCode.SCHEMA_INVALID_ATTRIBUTE,
+  ErrorCode.MISSING_REFERENCE_TARGET
```

Comment:
Minor comment: may be add a comma at the end of last line - the code should be auto re-formatted to have each error code on its own line (same for `packages/core/src/render3/error_code.ts` file).
```suggestion
  ErrorCode.MISSING_REFERENCE_TARGET,
```

Reply:
That was my thought but linting did the opposite (converted them to individual lines) - I can override though, maybe my personal settings are configured incorrectly

Reply:
The linting behavior looks correct, but adding an extra comma after the last element typically forces linter to put values on their own lines :)